python/metatensor_torch/metatensor/torch/atomistic/ase_calculator.py

```diff
@@ -321,6 +325,8 @@ def calculate(
         with record_function("ASECalculator::prepare_inputs"):
             outputs = _ase_properties_to_metatensor_outputs(properties)
             outputs.update(self._additional_output_requests)
+            if calculate_energy and self._model_has_energy_uncertainty:
+                outputs["energy_uncertainty"] = _get_energy_uncertainty_output()
```

Comment:
Is this something we always want to do? I see a point for having it on by default, but I feel like users should be able to turn this off

Reply:
I tend to agree, it's good to have it on-by-default, but it will have a non-zero overhead so maybe one should be able to switch it off